cache.js

```diff
@@ -54,12 +54,25 @@ let put = function (key, value) {
  * @param {string} key key whose mapping is to be removed from the map
  * @returns {(*|null)} the previous value associated with key or null if there was no mapping for key
  */
-let remove = function (key) {
+const remove = function (key) {
     return cache.remove(key);
 }
 
+/**
+ * Checks whether the mapping for a key from this map exists.
+ * 
+ * @memberof cache
+ * @param {string} key key whose mapping is to be checked in the map
+ * @returns {boolean} whether the key has a mapping therefore whether it exists
+ */
+const exists = function (key) {
+    return (!get(key, false) === false) ? true : false;
```

Comment:
The corner case here would be if the cache value is a boolean, and is set to false, then calling `exist` would always return false ? Maybe this should use `null` instead of a boolean in the default supplier ?

Reply:
err scratch my last comment, i'm not sure i'm  following the `!get(key, false) === false` logic.  `get` would return null here if the key does not exist?  Whats the benefit of calling `get(key, false)` over just `get(key)`?

Reply:
There is no benefit, I just did not think about the null option.
I have updated my PR.